Userland/Utilities/lzcat.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2023, Tim Schumacher <timschumi@gmx.de>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibArchive/Lzma.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/File.h>
+#include <LibMain/Main.h>
+
+ErrorOr<int> serenity_main(Main::Arguments arguments)
+{
+    StringView filename;
+
+    Core::ArgsParser args_parser;
+    args_parser.add_positional_argument(filename, "File to decompress", "file");
+    args_parser.parse(arguments);
+
+    NonnullOwnPtr<Stream> stream = TRY(Core::File::open(filename, Core::File::OpenMode::Read));
+    stream = TRY(Archive::Lzma::create(move(stream)));
+
+    // Arbitrarily chosen buffer size.
+    Array<u8, 4096> buffer;
```

Comment:
Use PAGE_SIZE?

Reply:
`PAGE_SIZE` is not consteval on Lagom, so we can't use it here. It also isn't nearly as important to align to pages compared to heap allocations. In this case, it really is just a nice and round number.